src/OpenColorIO/GpuShader.cpp

```diff
@@ -228,20 +245,20 @@ class PrivateImpl
 
     void add3DTexture(const char * textureName,
                       const char * samplerName,
-                      unsigned dimension,
+                      unsigned edgelen,
                       Interpolation interpolation,
                       const float * values)
     {
-        if(dimension > get3dLutMaxDimension())
+        if(edgelen > get3dLutMaxLength())
```

Comment:
Renamed these to prevent confusions with the new "dimensions" definition. This is also more consistent with the exposed methods.